pkgs/development/libraries/pipewire/default.nix

```diff
@@ -33,16 +33,16 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "pipewire";
-  version = "0.3.7";
+  version = "0.3.9";
 
-  outputs = [ "out" "lib" "dev" "doc" ];
+  outputs = [ "out" "dev" "doc" ];
```

Comment:
There seems to some sort cycle forming if `lib` is an output.

Reply:
See https://github.com/NixOS/nixpkgs/pull/93725/files#diff-cdeb3c763b3e2a37c5abe1b54fa8290eR54